thirdpartyemailpassword/docs/common-customizations/handling-signup-success.md

```diff
@@ -79,7 +79,7 @@ Please refer to the <a href="/docs/auth-react/emailpassword/callbacks#onhandleev
 
 ## 3) Handling sign up event on the backend
 
-### `handlePostSignUp`
+For this, you'll have to override signInUpPOST api of the ThirdPartyEmailPassword recipe.
```

Comment:
Isn't the content of this page now the same as that of handling-signin-success? If yes, then you can remove one of them like how it's done in thirdparty